src/server/pps/cmds/cmds.go

```diff
@@ -1094,11 +1095,12 @@ All jobs created by a pipeline will create commits in the pipeline's output repo
 	}
 	commands = append(commands, cmdutil.CreateAlias(listSecret, "list secret"))
 
+	var seed int64
 	runLoadTest := &cobra.Command{
-		Use:   "{{alias}}",
+		Use:   "{{alias}} <spec> ",
```

Comment:
maybe include "path" or "file" to indicate the spec is not literally passed as an argument, and that the command will read the filesystem.  There is a way to hide cobra commands with `Hidden`, maybe we should do that for the load testing stuff.

Reply:
Sure, I'll change it to `spec-file`. We don't want to hide the load testing. I could see users wanting to use it for a sanity check on their deployment, or the customer team for testing purposes.